src/model/entity/room.go

```diff
@@ -1,10 +1,82 @@
 package entity
 
+import (
+	"errors"
+	"math/rand"
+)
+
+type RoomType uint
+
+const (
+	RoomTypeStart RoomType = iota
+	RoomTypeOrdinary
+	RoomTypeFinish
+)
+
+const (
+	RoomMinHeight = 4
+	RoomMinWidth  = 4
+	RoomWalls     = 2
```

Comment:
Не совсем понятно что означает это значение, пока не посмотришь контекст использования по коду. Для большей ясности, можно наименовать вот так:

```suggestion
	RoomWallsNum     = 2
```